src/main/java/racingcar/validation/RoundValidation.java

```diff
@@ -0,0 +1,33 @@
+package racingcar.validation;
+
+public class RoundValidation {
+    public static final String ERROR_GAP = "[ERROR] 공백 없이 입력해주세요.";
+
+    public static final String ERROR_ONLY_INTEGER = "[ERROR] 정수만 입력해주세요.";
+
+    public static final String ERROR_ONE_MORE = "[ERROR] 1이상으로 입력해주세요.";
+
+    public void validateRoundAll(String userInput) {
+        validateGap(userInput);
+        validateOnlyInteger(userInput);
+        validateNumberOfRoundLimit(userInput);
+    }
+
+    public void validateGap(String userInput) {
+        if (userInput.contains(" ")) {
+            throw new IllegalArgumentException(ERROR_GAP);
+        }
+    }
+
+    public void validateOnlyInteger(String userInput) {
+        if (!userInput.matches("-?\\d+")) {
```

Comment:
음수에 대해서도 True를 반환하기 때문에 
```suggestion
        if (!userInput.matches("\\d+")) {
```
가 사용자 요건에 맞는 조건일 것 같습니다.